### ISSUE TITLE
Falta la vacuna CanSino en el Total de segunda dosis y en las regiones

### ISSUE THREAD

**Issue by AntoineBraultChile** [MinCiencia/Datos-COVID19#1005]
Hola,

Me parece que en el archivo Datos-COVID19/output/producto76/fabricante.csv, la vacuna CanSino  no se incluye en el total.
Tambien en el archivo https://github.com/MinCiencia/Datos-COVID19/blob/master/output/producto76/vacunacion.csv, los dosis por region y el total no incluyen CanSino.
No he verificado en los otros archivos si hay el mismo problema.

Gracias por la ayuda.
Saludos.

**Comment by MinCiencia** [comment]
hola, el producto 76 vacunación.csv si incluye la información, para fabricante estas mirando un producto obsoleto, por favor utiliza https://github.com/MinCiencia/Datos-COVID19/commits/master/output/producto83

**Comment by AntoineBraultChile** [comment]
Entonces hay que marcar que el producto 76 es obsoleto para que la gente no sea engañada. 
Porque las dosis de CanSino faltan en el total de las segundas dosis inyectadas.

**Comment by MinCiencia** [comment]
En realidad, no esta obsoleto, asi que lo mantendremos sin la marca que solicitas. Acabamos de revisar a raiz de tu pregunta y se continua actualizando gracias al trabajo de @juancri. 

Es bastante claro en los productos que revisaste que hay una columna especifica para vacunas de dosis unica, muchas gracias.

**Comment by AntoineBraultChile** [comment]
Ok gracias porque @juancri annucio hace 2 días el fin del soporte del repo sobre la vacunacion: https://github.com/juancri/covid19-vaccination/issues/34

**Comment by MinCiencia** [comment]
Lo anotaremos en este repo también así, sugiriendo visitar los productos que corresponda. Gracias.

**Comment by alonsosilvaallende** [comment]
Estamos de acuerdo en que en el producto 83 no está por fabricante, no? Veo sólo por establecimiento. Soy el único?

**Comment by dna33** [comment]
Cierto, ver issue que abrió en paralelo, pero está relacionado, que puso  @dafernandez75 

https://github.com/MinCiencia/Datos-COVID19/issues/1008
